source/common/http/match_wrapper/config.h

```diff
@@ -1,15 +1,96 @@
 #pragma once
 
 #include "envoy/extensions/common/matching/v3/extension_matcher.pb.validate.h"
+#include "envoy/extensions/filters/common/matcher/action/v3/skip_action.pb.h"
+#include "envoy/matcher/matcher.h"
 #include "envoy/server/filter_config.h"
 
+#include "source/common/http/matching/data_impl.h"
+#include "source/common/matcher/matcher.h"
 #include "source/extensions/filters/http/common/factory_base.h"
 
 namespace Envoy {
 namespace Common {
 namespace Http {
 namespace MatchWrapper {
 
+class SkipAction : public Matcher::ActionBase<
+                       envoy::extensions::filters::common::matcher::action::v3::SkipFilter> {};
+
+class DelegatingStreamFilter : public Envoy::Http::StreamFilter {
```

Comment:
Should we rename this file as well to better match this new filter name?

Reply:
Do you mean update it to `deligating.h`?